helm/chaos-mesh/templates/chaos-daemon-deployment.yaml

```diff
@@ -48,8 +52,11 @@ spec:
               mountPath: /run/containerd/containerd.sock
               {{- end }}
           ports:
-            - containerPort: {{ .Values.chaosDaemon.port }}
-              hostPort: {{ .Values.chaosDaemon.port }}
+            - name: grpc
+              containerPort: {{ .Values.chaosDaemon.grpcPort }}
+              hostPort: {{ .Values.chaosDaemon.grpcPort }}
+            - name: http
+              containerPort: {{ .Values.chaosDaemon.httpPort }}
```

Comment:
Did we also need hostPort here

Reply:
Yes, Prometheus needs to scrape the http port for metrics

Reply:
ok